pkg/ocicni/ocicni.go

```diff
@@ -476,15 +476,15 @@ func (plugin *cniNetworkPlugin) GetPodNetworkStatus(podNetwork PodNetwork) ([]cn
 func (network *cniNetwork) addToNetwork(cacheDir string, podNetwork *PodNetwork, ifName string, runtimeConfig RuntimeConfig) (cnitypes.Result, error) {
 	rt, err := buildCNIRuntimeConf(cacheDir, podNetwork, ifName, runtimeConfig)
 	if err != nil {
-		logrus.Errorf("Error adding network: %v", err)
+		log().Errorf("adding network: %v", err)
 		return nil, err
 	}
 
 	netconf, cninet := network.NetworkConfig, network.CNIConfig
-	logrus.Infof("About to add CNI network %s (type=%v)", netconf.Name, netconf.Plugins[0].Network.Type)
+	log().Debugf("adding CNI network %s (type=%v)", netconf.Name, netconf.Plugins[0].Network.Type)
```

Comment:
@saschagrunert can we change this one back to Infof() as well?

Reply:
Yes, I also rebased on top of the latest master to resolve the conflicts.